android/KMAPro/kMAPro/src/main/java/com/keyman/android/BannerController.java

```diff
@@ -0,0 +1,29 @@
+package com.keyman.android;
+
+import android.content.Context;
+
+import com.keyman.engine.KMManager;
+import com.keyman.engine.util.FileUtils;
+
+import java.io.File;
+
+public class BannerController {
+
+  // Paths relative to assets folder for banner themes
+  public static final String KM_BANNER_DIR = "banner";
+  public static final String KM_BANNER_THEME_KEYMAN = KM_BANNER_DIR + "/banner.html";
```

Comment:
suggest this should just be:

```suggestion
  public static final String KM_BANNER_THEME_KEYMAN = "banner.html";
```